integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DeployUtil.java

```diff
@@ -236,11 +236,11 @@ private static void createDeployJob(String deployScriptConfigMap, String namespa
    * @param password admin server password
    * @param cluster name of the cluster to deploy application
    * @param archivePath local path of the application archive
-   * @param hostHeader name of the cluster to deploy application
+   * @param hostHeader Host header for the curl command 
    * @param appName name of the application
    * @return ExecResult 
    */
-  public static ExecResult deployUsingRest(String host, String port,
+  public static ExecResult deployToClusterUsingRest(String host, String port,
```

Comment:
do we need this method deployToClusterUsingRest as there is deployUsingRest which can take cluster or server or combination for targets? If deployToClusterUsingRest is used, can that be modified to use deployUsingRest and remove deployToClusterUsingRest method?

Reply:
I kept the method deployToClusterUsingRest as convenience method with simple target syntax ( as cluster name )  as most of the IT class deploy the application to cluster.

Reply:
ok, can you refactor the code in these methods to remove duplicate code.

Reply:
refactored the code  as follows  ...
 public static ExecResult deployToClusterUsingRest(String host, String port,
            String userName, String password, String cluster,
            Path archivePath, String hostHeader, String appName) {
    String target = "{identity: [clusters,'" + cluster + "']}";
    return deployUsingRest(host, port, userName, password, target, archivePath, hostHeader, appName);
    }
 public static ExecResult deployToServerUsingRest(String host, String port,
            String userName, String password, String server,
            Path archivePath, String hostHeader, String appName) {
    String target = "{identity: [servers,'" + server + "']}";
    return deployUsingRest(host, port, userName, password, target, archivePath, hostHeader, appName);
  }

Reply:
looks good, pls update with latest jenkins results when you them